### PR TITLE
Tidy listenbrainz scrobbler

### PR DESCRIPTION
**Pull request by inverse**
**Describe the changes you made**

- Remove unused code
- Make FF case conditional (there was a console  error on chrome when I wasn't logged in)

**Additional context**

### CODE REVIEW

File: src/core/scrobbler/listenbrainz/listenbrainz-scrobbler.ts [hunk 1]
```diff
@@ -21,8 +21,6 @@ const apiUrl = `${baseUrl}/submit-listens`;
 export default class ListenBrainzScrobbler extends BaseScrobbler<'ListenBrainz'> {
 	public userApiUrl!: string;
 	public userToken!: string;
-	/** Can't find where this value is being set from */
-	private authUrl!: string;
```
Comment (inverse) [inline src/core/scrobbler/listenbrainz/listenbrainz-scrobbler.ts:25]: I think it was changed in https://github.com/web-scrobbler/web-scrobbler/commit/9f885e4c4d31d03e64b497b68f9b3a3e901ea1e2#diff-e2b6cac4e0d320435974b23d0ebc5174fb22df607c8def762f90dd7eb1004830L272 and never cleaned up

File: src/core/scrobbler/listenbrainz/listenbrainz-scrobbler.ts [hunk 2]
```diff
@@ -346,7 +335,9 @@ export default class ListenBrainzScrobbler extends BaseScrobbler<'ListenBrainz'>
 		// NOTE: Use 'same-origin' credentials to fix login on Firefox ESR 60.
 		const promise = fetch(url, {
 			method: 'GET',
+			// #v-ifndef VITE_FIREFOX
```
Comment (yayuyokitano) [inline src/core/scrobbler/listenbrainz/listenbrainz-scrobbler.ts:338]: ifndef is for every browser except for firefox, i think you mightve meant ifdef?

Comment (inverse) [reply src/core/scrobbler/listenbrainz/listenbrainz-scrobbler.ts]: Good catch :see_no_evil:

**Comment by yayuyokitano** [comment]
Still works on my end and cant see how its used so LGTM

**Pull request merged**